content/02.introduction.md

```diff
@@ -8,23 +8,25 @@ Integrating functional genomics data and GWAS data [@doi:10.1016/j.ajhg.2018.04.
+Moreover, since we rely on a large and heterogeneous RNA-seq dataset, we were also able to identify modules associated with cell types under specific stimuli or disease states.
 We replicated gene module associations with cardiovascular and autoimmune diseases in the Electronic Medical Records and Genomics (eMERGE) network phase III [@doi:10.1038/gim.2013.72].
-Moreover, we performed a CRISPR-screen to analyze lipid regulation in HepG2 cells and observed more robust trait associations with modules than with individual genes, even when single genes known to be involved in lipid metabolism did not reach genome-wide significance.
+Furthermore, we performed a CRISPR screen to analyze lipid regulation in HepG2 cells and observed more robust trait associations with modules than with individual genes, even when single genes known to be involved in lipid metabolism did not reach genome-wide significance.
 <!-- Since our approach incorporates groups of genes associated with a phenotype instead single genes, it was also more robust in finding meaningful gene module-trait associations, even when individual genes involved in lipid metabolism did not reach genome-wide significance in lipid-related traits. -->
 Compared to a single-gene approach, our module-based method also better predicted FDA-approved drug-disease links by capturing tissue-specific pathophysiological mechanisms linked with the mechanism of action of drugs (e.g., niacin with cardiovascular traits via a known immune mechanism), suggesting that modules may provide a better means to examine drug-disease relationships than individual genes.
 Finally, exploring the phenotype-module space also revealed stable trait clusters associated with relevant tissues, including a complex branch involving lipids with cardiovascular, autoimmune, and neuropsychiatric disorders.
-In summary, instead of considering single genes associated with different complex traits, PhenoPLIER incorporates groups of genes that act together to carry out different functions in specific cell types, and this improves robustness to detect and interpret genetic associations.
-The approach can aid future endeavors focused on identifying potential drug targets for complex disease phenotypes.
+In summary, instead of considering single genes associated with different complex traits, PhenoPLIER incorporates groups of genes that act together to carry out different functions in specific cell types.
+This improves robustness to detect and interpret genetic associations, and here we show how it can identify alternative and potentially more promising drug targets when single gene associations are not detected or are not druggable.
```

Comment:
I think "promising drug targets when single gene associations are not detected or are not druggable" is a pretty far bridge. We don't actually target the genes from the CRISPR screen. What do you think about considering these as functionally validated or similar? Functionally important?

Reply:
What about this? I also changed the word `identify` to `prioritize`, since the first one is too strong.

```suggestion
This improves robustness to detect and interpret genetic associations, and here we show how it can prioritize alternative and potentially more promising drug targets when single gene associations do not capture all functionally important hits.
```

Reply:
Are the genes found actually drug targets though? That was the bit I'm not sure about.